sap_hana/datadog_checks/sap_hana/data/conf.yaml.example

```diff
@@ -29,7 +29,7 @@ instances:
     #
   - server: <SERVER>
 
-    ## @param port - integer - optional - default: 30015
+    ## @param port - integer - optional - default: 39015
```

Comment:
The current default port used in the integration seems to be `30015` https://github.com/DataDog/integrations-core/blob/c7f0abe159a7b72d0c24356b1c8e2cbaa2cbb506/sap_hana/datadog_checks/sap_hana/sap_hana.py#L33
Maybe the port we're using should be updated though. @ofek WDYT?

Reply:
Thanks for the info + looking into this @coignetp! We received word from support that "there is a default port setting with port: 39015 in the yaml that differs from what we have in the docs", and it seems to be the same on my end within the available Agent checks. Also seeing it within the Agent GUI listed as the default, too. 🧐 Might be the other way around though?

Reply:
The doc said `39015` before, but it was changed and shipped in 7.24.0 (released ~ last week) because it didn't match what was in the python code

Reply:
https://help.sap.com/viewer/ports search for `SAP HANA Platform`

Reply:
![Capture d’écran 2020-12-08 à 15 17 24](https://user-images.githubusercontent.com/26168335/101495148-7ce9a100-3968-11eb-927f-c05024b28ada.png)
Seems that `30015` is indeed the default port then

Reply:
Thanks for looking further into this all! I'll go ahead and remove this file as it doesn't require any changes.